conf_mesa.sh

```diff
@@ -39,9 +40,9 @@ else
 fi
 
 
-rm -r build$1
+rm -r build${buildtype}$1
```

Comment:
I'd like to keep the option of using "build" and "build32" regardless of the build type.

Reply:
I removed this part of the change.